app/stores/toplevel/ProfileStore.js

```diff
@@ -125,7 +126,7 @@ export default class ProfileStore extends Store {
 
   // ========== Current/Custom Theme ========== //
 
-  @computed get currentTheme(): string {
+  @computed get currentTheme(): Themes {
```

Comment:
Maybe this one should be Theme in singular

Reply:
sure why not